docs/developer/Technical Documentation/Dev Process/Dev-flow_git-diagram.md

```diff
@@ -0,0 +1,69 @@
+# Dev flow (git diagram)
```

Comment:
it best if you change directly to the new file structure, mentioned [here](https://github.com/eclipse-tractusx/sig-release/issues/805) to be TRG compliant, I suggest to a admin directory instead to developer/Technical Documentation

Reply:
addressed with: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/1/commits/42d6fbbbed7602a02db3de9e6ba3082b30686e22